conf/livy.conf

```diff
@@ -38,3 +38,7 @@
 # default it's empty, meaning users can only reference remote URIs when starting their
 # sessions.
 # livy.file.local-dir-whitelist =
+
+# Whether to enable csrf protection, by default it is false. If it is enabled, client should add
+# http-header "X-Requested-By" in request if the http method is POST/DELETE/PATCH.
```

Comment:
I think both PATCH/PUT are not used in livy api. But I just notice they are defined in `JsonServlet` and only be used in `TestJsonServlet`. So I will just keep them here in case they will be used in api in future.